assets/wizards/setup/views/welcome/index.js

```diff
@@ -166,7 +187,13 @@ const Welcome = ( { buttonAction } ) => {
 			return __( 'Installation error', 'newspack' );
 		}
 		if ( isInit ) {
-			return __( 'Welcome to WordPress for your Newsroom!', 'newspack' );
+			return (
+				<>
+					{ __( 'Welcome to Newspack,', 'newspack' ) }
+					<br />
+					{ __( 'The platform for News.', 'newspack' ) }
+				</>
+			);
```

Comment:
@thomasguillot – the "Welcome to Newspack…" header looks odd to me – shouldn't it be centered? And the "The" should rather be lowercased, since it comes after a comma.

<img width="626" alt="image" src="https://user-images.githubusercontent.com/7383192/139658275-1b0298ec-b004-457b-b4cb-494a146528b1.png">

Reply:
Fixed here: https://github.com/Automattic/newspack-plugin/pull/1077/commits/db9fae2a99ee1a99e122d020eb5223e358c42694